web-client/integration-tests/docketClerkCaseInventoryReport.test.ts

```diff
@@ -127,60 +140,67 @@ describe('case inventory report journey', () => {
     await cerebralTest.runSequence('addCaseToTrialSessionSequence');
   });
 
-  it('get the updated case inventory counts', async () => {
+  it('should navigate to the case inventory report and check for new case', async () => {
     await refreshElasticsearchIndex();
 
-    //New (+1 from initial)
     await cerebralTest.runSequence('openCaseInventoryReportModalSequence');
     await cerebralTest.runSequence('updateScreenMetadataSequence', {
       key: 'status',
       value: CASE_STATUS_TYPES.new,
     });
     await cerebralTest.runSequence('submitCaseInventoryReportModalSequence');
-    let updatedCaseInventoryCount = cerebralTest.getState(
+    const updatedCaseInventoryCount = cerebralTest.getState(
       'caseInventoryReportData.totalCount',
     );
 
     expect(updatedCaseInventoryCount).toEqual(
-      initialCaseInventoryCounts.new + 1,
+      initialCaseInventoryCounts.new! + 1,
     );
-    //New, Judge Colvin (same as initial)
+  });
+
+  it('should get the updated new case inventory count for Judge Colvin', async () => {
     await cerebralTest.runSequence('updateScreenMetadataSequence', {
       key: 'associatedJudge',
       value: 'Colvin',
     });
     await cerebralTest.runSequence('submitCaseInventoryReportModalSequence');
-    updatedCaseInventoryCount = cerebralTest.getState(
+    const updatedCaseInventoryCount = cerebralTest.getState(
```

Comment:
What was the cause of flakiness with reassigning this variable? Is that something to be aware of in our other tests?

Reply:
I don't think that reassigning the variable was causing flakiness directly, it just wasn't necessary after splitting the one giant test into smaller ones